src/frontend/app/shared/components/list/list-types/services-wall/service-instance-card/service-instance-card.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { BehaviorSubject } from 'rxjs/BehaviorSubject';
+
+import { IServiceInstance } from '../../../../../../core/cf-api-svc.types';
+import { ServicesWallService } from '../../../../../../features/services/services/services-wall.service';
+import { DeleteServiceBinding, DeleteServiceInstance } from '../../../../../../store/actions/service-instances.actions';
+import { AppState } from '../../../../../../store/app-state';
+import { APIResource } from '../../../../../../store/types/api.types';
+import { AppChip } from '../../../../chips/chips.component';
+import { MetaCardMenuItem } from '../../../list-cards/meta-card/meta-card-base/meta-card.component';
+import { CardCell } from '../../../list.types';
+
+@Component({
+  selector: 'app-service-instance-card',
+  templateUrl: './service-instance-card.component.html',
+  styleUrls: ['./service-instance-card.component.scss'],
+  providers: [
+    ServicesWallService
+  ]
+})
+export class ServiceInstanceCardComponent extends CardCell<APIResource> implements OnInit {
+  cfGuid: string;
+  cardMenu: MetaCardMenuItem[];
+
+  @Input('row') row: APIResource<IServiceInstance>;
```

Comment:
Strictly speaking you shouldn't need to declare row, as it comes in already typed via `CardCell` -->`TableCellCustom` `row`